src/software/Localization/main_SfM_Localization.cpp

```diff
@@ -93,7 +93,7 @@ int main(int argc, char **argv)
     << "  contains a single intrinsics and, if so, take this value as intrinsics for the query images.\n"
     << "  (OFF by default)\n"
     << "[-e|--export_structure] (switch) when switched on, the program will also export structure to output sfm_data.\n"
```

Comment:
Can you motivate why you have chose to set the structure as an option?
Having the structure in all the case can be desired...